core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/install/ClusterManagerConfigureKerberosHandler.java

```diff
@@ -0,0 +1,50 @@
+package com.sequenceiq.cloudbreak.reactor.handler.cluster.install;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.core.cluster.ClusterBuilderService;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.install.ClusterManagerConfigureKerberosFailed;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.install.ClusterManagerConfigureKerberosRequest;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.install.ClusterManagerConfigureKerberosSuccess;
+import com.sequenceiq.cloudbreak.service.CloudbreakException;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.EventHandler;
+
+import reactor.bus.Event;
+import reactor.bus.EventBus;
+
+@Component
+public class ClusterManagerConfigureKerberosHandler implements EventHandler<ClusterManagerConfigureKerberosRequest> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClusterManagerConfigureKerberosHandler.class);
+
+    @Inject
```

Comment:
I would use constructor injection instead of field since this is the suggested way - by the spring guys - of doing it